src/openforms/authentication/api/views.py

```diff
@@ -62,3 +70,38 @@ def delete(self, request, *args, **kwargs):
 
         request.session.flush()
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class SubmissionLogoutView(GenericAPIView):
+    authentication_classes = ()
+    permission_classes = (ActiveSubmissionPermission,)
+    serializer_class = (
+        serializers.Serializer
+    )  # just to shut up some warnings in drf-spectacular
+
+    queryset = Submission.objects.all()
+    lookup_field = "uuid"
+
+    @extend_schema(
+        operation_id="submission_session_destroy",
+        summary=_("Delete session"),
```

Comment:
```suggestion
        summary=_("Delete form session"),
```
technically it's a submission session, but in all the other language we use "form session".